crates/re_data_ui/src/component.rs

```diff
@@ -78,7 +81,9 @@ impl DataUi for EntityComponentWithInstances {
                         ui.label("Instance Key");
                     });
                     header.col(|ui| {
-                        ui.label(self.component_name().short_name());
+                        // TODO(jleibs): fix short_name
+                        //ui.label(self.component_name().short_name());
```

Comment:
The fact that we're now using a type-alias for ComponentName instead of a trait keeps us from implementing these conveniently with a trait :-(